src/test/java/com/gui/LoginPageTest.java

```diff
@@ -6,44 +6,41 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-public class LoginPageTest extends AbstractTest{
+public class LoginPageTest extends AbstractTest {
 
     @Test(groups = "group13")
-    public void LoginTest(){
-        HomePage homePage =new HomePage(getDriver());
-        LoginPage loginPage = homePage.getLoginPage();
+    public void testLoginFunctionality() {
+        HomePage homePage = new HomePage(getDriver());
+        LoginPage loginPage = homePage.openLoginPage();
         loginPage.enterLogin("abcdee");
         loginPage.enterPassword("Z6FBty@R2N4NWN2");
         loginPage.submitCredentials();
-        Assert.assertTrue(loginPage.isUserOnMyAccountPage(),"User is not on My Account page");
+        Assert.assertTrue(loginPage.isUserOnMyAccountPage(), "User is not on My Account page");
     }
 
     @Test(groups = "group1")
-    public void IncorrectLoginTest() throws InterruptedException {
-        HomePage homePage =new HomePage(getDriver());
-        LoginPage loginPage = homePage.getLoginPage();
+    public void testIncorrectLogin() throws InterruptedException {
+        HomePage homePage = new HomePage(getDriver());
+        LoginPage loginPage = homePage.openLoginPage();
         loginPage.enterLogin("abcdee");
         loginPage.submitCredentials();
-        Assert.assertTrue(loginPage.isALoginErrorPresent(),"Login error is not present");
+        Assert.assertTrue(loginPage.isALoginErrorPresent(), "Login error is not present");
     }
 
     @Test(groups = "group13")
-    public void RegisterTest(){
+    public void testUserRegistration() {
         HomePage homePage = new HomePage(getDriver());
-        LoginPage loginPage = homePage.getLoginPage();
+        LoginPage loginPage = homePage.openLoginPage();
         RegisterPage registerPage = loginPage.clickRegisterButton();
-        registerPage.enterFirstName("John");
-        registerPage.enterLastName("Doe");
-        registerPage.enterEmail("exeample@mail.com");
-        registerPage.enterAdress1("Some Street 5/43");
-        registerPage.enterCity("Aberdeen");
-        registerPage.chooseRegion();
-        registerPage.enterZipCode("AB10");
-        registerPage.enterLogin("Jonhjohne");
-        registerPage.enterPassword("Z6FBty@R2N4NWN2");
-        registerPage.enterConfirmPassword("Z6FBty@R2N4NWN2");
-        registerPage.checkPrivacyPolicy();
-        registerPage.submitRegistration();
-        Assert.assertTrue(registerPage.isAccountCreatedTextPresent(),"Account has not been created");
+        registerPage.fillOutRegistrationForm(
```

Comment:
Create a User object instead of passing 8+ parameters, create a helper method to construct the User object